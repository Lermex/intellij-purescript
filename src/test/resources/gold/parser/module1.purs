module Main2 where