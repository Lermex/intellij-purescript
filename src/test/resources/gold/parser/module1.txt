Purescript File
  PSASTWrapperElement(Program)
    PSASTWrapperElement(Module)
      PsiElement(module)('module')
      PsiElement(whitespace)(' ')
      PSASTWrapperElement(ModuleName)
        PSASTWrapperElement(Qualified)
          PsiElement(proper name)('Main')
        PsiElement(whitespace)(' ')
      PsiElement(where)('where')