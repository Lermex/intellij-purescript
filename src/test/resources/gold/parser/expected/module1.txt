Purescript File
  PSDeclarationImpl(DECLARATION)
    PSModuleDeclarationImpl(MODULE_DECLARATION)
      PsiElement(PSTokenType.module)('module')
      PSModuleNameImpl(MODULE_NAME)
        PsiElement(PSTokenType.PROPER_NAME)('Main')
      PsiElement(PSTokenType.where)('where')